kevm-pyk/src/tests/integration/test_prove.py

```diff
@@ -329,7 +329,6 @@ def test_prove_rules(
     tmp_path: Path,
     caplog: LogCaptureFixture,
     no_use_booster: bool,
-    force_sequential: bool,
```

Comment:
I guess these changes are subsumed here: https://github.com/runtimeverification/evm-semantics/pull/2608

And can be removed from this PR.

Reply:
They will go away, yes.